imbi/sonarqube.py

```diff
@@ -121,4 +122,62 @@ async def create_project(self, project: 'automations.Project', *,
                         'failed to rename main branch %s for %s: %s',
                         branch['name'], project_key, response.code)
 
+        if await self._is_gitlab_alm_available():
+            self.logger.debug('checking for PR decoration on %s', project_key)
+            response = await self.api(
+                yarl.URL('/alm_settings/get_binding').with_query({
+                    'project': project_key,
+                })
+            )
+            if response.code == 404:  # not configured or doesn't exist
```

Comment:
Personal preference but I feel like the 404 response is self explanatory and the comment doesn't add much value.

Reply:
slightly interesting thing here ... usually a 404 means that the thing that you requested does not exist.  SonarQube returns a 404 here when the binding exists but it hasn't been configured.  I was expecting a 404 when the project didn't exist and some type of an empty response when it was unconfigured.  Unfortunately the [SonarQube Web API docs](https://next.sonarqube.com/sonarqube/web_api/api/alm_settings/get_binding) are less than clear about the response in this case.

Reply:
In that case, I think the extra context can be helpful here